source/patterns/@aws-solutions-constructs/aws-fargate-kinesisfirehose/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@aws-solutions-constructs/aws-fargate-kinesisfirehose",
+  "version": "2.29.0",
```

Comment:
In this file, there are many explicit version numbers here and din the dependency sections below. Most of these should be set to 0.0.0, like [here](https://github.com/awslabs/aws-solutions-constructs/blob/main/source/patterns/%40aws-solutions-constructs/aws-fargate-kinesisstreams/package.json), correct?